Kajabi/sage-lib
# Property - moved spacing to the right to resolve alignment on mobile

## Description

- [x] move margin to the right to fix mobile alignment issue

## Screenshots

|  Before  |  After  |
|--------|--------|
|![Screen Shot 2021-09-28 at 4 09 54 PM](https://user-images.githubusercontent.com/1241836/135168978-b2a43316-329f-49f1-a788-9a63b28ab684.png)|![Screen Shot 2021-09-28 at 4 10 05 PM](https://user-images.githubusercontent.com/1241836/135168985-35996ec1-40a6-4f40-90dd-536551e83567.png)|


## Testing in `sage-lib`

- Visit the sage property component page and view at a screen width of `320px`
- Verify that all property items are left aligned


## Testing in `kajabi-products`

1. (**LOW**) Moved margins to the right to correct mobile alignment.
   - [ ] Coaching Program Session view.


## Related
